source/main/plugins_d.cfg.in

```diff
@@ -1,15 +1,16 @@
 # Defines plugins to load
+# NOTE: OGRE13 adds the _d suffixes itself!
```

Comment:
https://github.com/OGRECave/ogre/blob/master/Docs/1.12-Notes.md#automatic-plugin-discovery-for-windows-debug-builds